lib/utils/httpRequestor.js

```diff
@@ -102,56 +102,73 @@ exports.create = function(requestorConfig) {
   var postFile = (options, callback) => {
     var requestExtension = { body: getFileBody(options) };
 
-    return methodRequest(options, request.postAsync, 'POST', callback, requestExtension);
+    return methodRequest(options, request.post, 'POST', callback, requestExtension);
   };
 
   var deleteFunc = (options, callback)  =>
-    methodRequest(options, request.delAsync, 'DELETE', callback);
+    methodRequest(options, request.delete, 'DELETE', callback);
 
   var put = (options, callback) => {
     var requestExtension = { body: JSON.stringify(options.body) };
 
-    return methodRequest(options, request.putAsync, 'PUT', callback, requestExtension);
+    return methodRequest(options, request.put, 'PUT', callback, requestExtension);
   };
 
   var methodRequest = (options, method, methodName, callback, requestExtension) => {
     var baseRequestOptions = {
       url: buildUrl(options),
       headers: buildHeaders(options),
-      qs: options.queryParameters,
-      encoding: options.encoding,
-      gzip: true
```

Comment:
Is the encoding type not ultimately configurable by the public api of this SDK? I think Axios already uses gzip by default so no concern on that one, but I want to make sure we're not removing `encoding` configurability from consumers.

Reply:
Yes from what I could see this was formally hard coded to `true` and not configurable. I just double checked and it axios does use gzip by default so we should be good here.

Reply:
Apologies if you're still looking at the first part of @joshualoehr question @mattmeyerink but it's the removal of `encoding: options.encoding` that is the larger concern here. This should be kept configurable as we can't guarantee it's not being used.

Reply:
Oh yes good catch! Updated this to the equivalent config for encoding in axios in the latest changes.